docs/reference-architectures/serverless/web-app.md

```diff
@@ -12,46 +12,34 @@ ms.custom: seodec18, serverless
 
 # Serverless web application on Azure
 
-This reference architecture shows a [serverless](https://azure.microsoft.com/solutions/serverless/) web application. The application serves static content from Azure Blob Storage, and implements an API using Azure Functions. The API reads data from Cosmos DB and returns the results to the web app.
+This reference architecture shows a [serverless](https://azure.microsoft.com/solutions/serverless/) web application. The application serves static content from Azure Storage, and implements an API using Azure Functions. The API reads data from Cosmos DB and returns the results to the web app.
 
 ![GitHub logo](../../_images/github.png) A reference implementation for this architecture is available on [GitHub][github].
 
 ![Reference architecture for a serverless web application](./_images/serverless-web-app.png)
 
-The term serverless has two distinct but related meanings:
```

Comment:
The term "serverless" means different things to everyone. I think what we're trying to do here is show what services that we have that fall under the serverless bucket.